CHANGES.md

```diff
@@ -1,5 +1,13 @@
 ### 3.0.0 (2016-soon!)
 
+* remove type `id` from `DEVICE` module type. (#612, by @yomimono and @talex5)
+* add to `RANDOM` a `generator` type, making it interoperable with `Nocrypto.Rng`. (#610, by @hannesm)
```

Comment:
the major thing was to revise the RANDOM signature in #551, #610 was only a minor thing on top of that